_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -0,0 +1,205 @@
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "cluster.routing.allocation.enable": "primaries"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : {
+       "cluster" : {
+         "routing" : {
+           "allocation" : {
+             "enable" : "primaries"
+           }
+         }
+       }
+     },
+     "transient" : { }
+   }
+   ```
+
+1. Perform a flush operation on the cluster to commit transaction log entries to the Lucene index:
+   ```json
+   POST "/_flush?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "_shards" : {
+       "total" : 4,
+       "successful" : 4,
+       "failed" : 0
+     }
+   }
+   ```
+1. Set the `remote_store.compatibility_mode` to `mixed` to allow remote-store backed nodes to join the cluster
+   ```json
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "remote_store.compatibility_mode": "mixed"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : { 
+      "remote_store" : {
+      "compatibility_mode" : "mixed"
+      },
+      "transient" : { }
+   }
+   }
+   ```
+2. Review your cluster and identify the first node to upgrade.
+2. Provide the remote store repository details as node attributes in `opensearch.yml`, as shown in the following example.
+
+   ```yml
+   # Repository name
+   node.attr.remote_store.segment.repository: my-repo-1
+   node.attr.remote_store.translog.repository: my-repo-2
+   
+   # Segment repository settings
+   node.attr.remote_store.repository.my-repo-1.type: s3
+   node.attr.remote_store.repository.my-repo-1.settings.bucket: <Bucket Name 1>
+   node.attr.remote_store.repository.my-repo-1.settings.base_path: <Bucket Base Path 1>
+   node.attr.remote_store.repository.my-repo-1.settings.region: us-east-1
+   
+   # Translog repository settings
+   node.attr.remote_store.repository.my-repo-2.type: s3
+   node.attr.remote_store.repository.my-repo-2.settings.bucket: <Bucket Name 2>
+   node.attr.remote_store.repository.my-repo-2.settings.base_path: <Bucket Base Path 2>
+   node.attr.remote_store.repository.my-repo-2.settings.region: us-east-1
+   
+   ```
+1. Stop the node you are migrating. Do not delete the volume associated with the container when you delete the container. The new OpenSearch container will use the existing volume. **Deleting the volume will result in data loss**.
+1. Deploy a new container running the same version of OpenSearch and mapped to the same volume as the container you deleted.
+1. Query the `_cat/nodes` endpoint after OpenSearch is running on the new node to confirm that it has joined the cluster. Wait for the cluster to become green again.
+1. Repeat steps 2 through 5 for each node in your cluster. 
+1. Reenable shard replication:
```

Comment:
```suggestion
1. Enable shard replication:
```

Reply:
We are reenabling as it was enabled before.